backend/tournesol/urls.py

```diff
@@ -113,6 +113,11 @@
         RateLaterList.as_view(),
         name="usersme_ratelater_list",
     ),
+    path(
+        "users/me/rate_later/<str:poll_name>/bulk_create",
```

Comment:
To disambiguate more clearly from URLs that contain an entity uid, I would suggest to prefix actions with `_`. This is also more consistent with the existing `"users/me/contributor_ratings/<str:poll_name>/_all/"`

```suggestion
        "users/me/rate_later/<str:poll_name>/_bulk_create",
```

Reply:
Done in b320048b1053c59713e73de460bd7059dd87b520.